src/datetime.cc

```diff
@@ -230,7 +230,42 @@ ularge julian_day_number(int d, int m, int y)
 //   Compute the Julian day number given day, month and year
 // ----------------------------------------------------------------------------
 {
-    uint rm = (m-14)/12;
+  // The algorithm below seems to come from:
```

Comment:
The indenting is not consistent with the rest of the files. I will fix it. I will also fix a couple of minor typos. Otherwise, this looks good. I will also shorten the comment, notably the part about the C++ standard not being consistent about integer divisions. AFAICT, C++ has always been consistent as rounding towards zero at least since 98 except for the overflow cases, and C has been consistent at least since 89. If you are aware of any relevant implementation that diverges from this behaviour, I am willing to make amends and put that part of the comment back 😄